src/routes/User/Users.ts

```diff
@@ -15,12 +15,16 @@ const User = userModel;
 /******************************************************************************
  *                      Get All Users - "GET /api/users/all"
  ******************************************************************************/
-/*
-router.get('/all', async (req: Request, res: Response) => {
-    const users = await User.find();
+
+router.get('', async (req: Request, res: Response) => {
+    const users = await User.find({});
     return res.status(OK).json({users});
 });
-*/
+
+router.get('/:id', async(req:Request, res: Response)=>{
+    const user = await User.find({_id: req.params.id});
+    return res.status(OK).json({user});
+});
```

Comment:
Would be good to add an error handling
```suggestion
router.get('/:id', async(req:Request, res: Response) => {
    try {
        const user = await User.find({_id: req.params.id});
        return res.status(OK).json({user});
    } catch (error) {
        return res.status(BAD_REQUEST).send(error.message);
    }
});
```

Reply:
Sure, my oversight.